backend/src/services/models.py

```diff
@@ -30,6 +30,7 @@ class ServiceBase(CoreModel):
     )
     tags: List[ExecutionUnitTag] | None = Field(sa_column=Column(JSON), default=None)
     url: AnyHttpUrl = Field(sa_column=Column(String))
+    docs_url: Optional[AnyHttpUrl] = Field(sa_column=Column(String))
```

Comment:
Could it be possible to write `docs_url: AnyHttpUrl | None = Field(sa_column=Column(String))`?